uint8 color_index
float64[] position_relative
float64[] normal_relative
float64 q
float64[] position